packages/site/tsconfig.base.json

```diff
@@ -2,6 +2,8 @@
     "extends": "../../tsconfig.core.json",
     "compilerOptions": {
         "target": "ES2015",
+        "module": "ESNext",
+        "moduleResolution": "Node",
```

Comment:
Used the same settings that we currently have in Demo site. Module resolution should probably be changed to "Bundler", but that requires an update to TS 5.5